rllib/agents/sac/sac_tf_policy.py

```diff
@@ -154,6 +155,8 @@ def _get_dist_class(config: TrainerConfigDict, action_space: gym.spaces.Space
     """
     if isinstance(action_space, Discrete):
         return Categorical
+    elif isinstance(action_space, Simplex):
```

Comment:
Why not give it the same name as Dirichlet?

Reply:
Simplex is the Space, Dirichlet the "matching" distribution.